### DISCUSSION TITLE
Codeium.vim plugin keymaps option doesn't seem to work properly

### DISCUSSION THREAD

**Discussion by Readf0x** [nix-community/nixvim#2134] [Q&A]
I'm gonna be honest, this might an issue with the codeium plugin itself. But the keymaps simply do not work, and the plugin's default keymaps still work, despite the default value of them supposedly being null according to the documentation.
my config looks like this:
```nix
codeium-vim = {
  enable = true;
  keymaps = {
    accept = "<C-a>";
  };
};
```
I checked my `init.lua`, and it should be set. However, not only does it not function, I don't see the original plugin's keymaps being unset anywhere.

**Comment by MattSturgeon** [comment]
For reference, our implementation is here: https://github.com/nix-community/nixvim/blob/main/plugins/completion/codeium-vim.nix

IDK if anything has changed upstream, but it looks like @GaetanLepage did some refactoring work to the keymaps around 7 months ago.

As a troubleshooting step, could you check if the "command" we have configured by default still works: https://github.com/nix-community/nixvim/blob/433673a7b8b439b77a35ca4db53becdd1a0d0c7d/plugins/completion/codeium-vim.nix#L10-L36

e.g. try running `:lua vim.fn['codeium#Clear']()`
